.gitignore

```diff
@@ -0,0 +1,62 @@
+nconfig.env
```

Comment:
First two lines have typos but they're not needed anyway as they're duplicated below